package.json

```diff
@@ -5,19 +5,27 @@
   "author": "Lucy Poulton <lucy@lucypoulton.net>",
   "license": "BSD-2-Clause",
   "repository": "github:lucypoulton/djs-slash-helper",
-  "files": ["dist/**"],
+  "files": [
+    "dist/**"
+  ],
   "type": "module",
   "main": "dist/index.js",
   "scripts": {
-    "build": "tsc"
+    "build": "tsc",
+    "postinstall": "tsc --outDir ./build"
```

Comment:
dunno the exact reason but when I did `yarn add github.com/knightzmc/djs-slash-helper` it wouldn't compile without this, presumably because the js files didn't exist or something

this ecosystem is so confusin

Reply:
the package is compiled prior to deployment, just going straight from source won't work